blscosi/blscosi/README.md

```diff
@@ -83,5 +83,7 @@ This will first contact each server individually and then check a few random col
 
 ## Further Information
 
+During the sign and verify operations, each message is prefixed with a constant token. Withtout it, the BLSCoSi service could be used as an oracle, allowing a malicious user to lead an attack by getting the service to sign a malicious block, for example.
```

Comment:
```suggestion
During the sign and verify operations, each message is prefixed with a constant token. Without it, the BLSCoSi service could be used as an oracle, allowing a malicious user to lead an attack by getting the service to sign a malicious block, for example.
```

Reply:
Also, please keep these lines to 80 characters, so that even if someone is not reading them via GitHub's markdown formatter, they are beautiful.